src/modules/awake/Awake/Awake.csproj

```diff
@@ -15,6 +15,9 @@
     <Company>Microsoft Corporation</Company>
     <Version>$(Version).0</Version>
     <ApplicationIcon>Images\Awake.ico</ApplicationIcon>
+    <SupportedOSPlatformVersion>10.0.17763.0</SupportedOSPlatformVersion>
```

Comment:
The current minimum we're supporting is 18362.

Reply:
Will also fix!